src/Microsoft.AspNet.JsonPatch/Adapters/IObjectAdapter.cs

```diff
@@ -8,14 +8,13 @@ namespace Microsoft.AspNet.JsonPatch.Adapters
     /// <summary>
     /// Defines the operations that can be performed on a JSON patch document.
     /// </summary>
-    /// <typeparam name="T"></typeparam>
-    public interface IObjectAdapter<T> where T : class
+    /// <typeparam name="TModel">The type of the model.</typeparam>
+    public interface IObjectAdapter<TModel> where TModel : class
```

Comment:
This is an improvement, but we should still think about where in code the `TModel` really even makes sense. Just because you have a type doesn't mean you have to be generic (if that makes sense).